cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -3,15 +3,85 @@
+============
 
 .. highlight:: console
 
 A Docker image with CDAP SDK pre-installed is available on the Docker Hub for download.
 
-To use the **Docker image**:
+To use the **Docker image**, you can either use Docker's :ref:`Kitematic
+<docker-kitematic>` (on Mac OS X and Windows), a graphical user interface for running
+Docker containers, or start the container from :ref:`a command line <docker-command-line>`.
+
+.. _docker-kitematic:
+
+Docker using Kitematic
+======================
+
+`Docker Kitematic <https://www.docker.com/docker-kitematic>`__ is available as part of the
+`Docker Toolbox <https://www.docker.com/docker-toolbox>`__ for either Mac OS X or Microsoft Windows.
+It is a graphical user interface for running Docker containers. Follow these steps to install 
+Kitematic and then download, start, and connect to an instance of CDAP.
+
+#. Download and install the `Docker Toolbox <https://www.docker.com/docker-toolbox>`__ for 
+   either Mac OS X or Microsoft Windows.
+
+#. Start Kitematic. On Mac OS X, it will be installed in ``/Applications/Docker/Kitematic``; on 
+   Windows, in ``Start Menu > Docker > Kitematic``.
+   
+#. Once Kitematic has started |---| you can either login to the Docker Hub or skip that step |---| 
+   search for the **CDAP image** using the search box at the top of the window. Then click on the
+   repository menu, circled in red here:
+ 
+     .. image:: ../../_images/kitematic/kitematic-1-searching.png
+        :width: 8in
+        :align: center
+
+#. Click on the tags button:
+ 
+     .. image:: ../../_images/kitematic/kitematic-2-tags.png
+        :width: 4in
+        :align: center
+
+#. Scroll the window to the last version shown, and select it (unless you'd like a different version).
+   Note that the tag **latest** is the last version that was put up at Docker Hub, which is not the 
+   necessarily the most-current version:
+ 
+     .. image:: ../../_images/kitematic/kitematic-3-select-tag.png
+        :width: 4in
+        :align: center
+
+#. Close the menu by pressing the ``X`` in the circle. Press "Create" to download and start the CDAP image. 
+   When it has started up, you will see in the logs a message that the CDAP UI is listening on port 9999:
+ 
+     .. image:: ../../_images/kitematic/kitematic-4-cdap-started.png
+        :width: 8in
+        :align: center
+
+#. To connect a web browser for the CDAP UI, you'll need to find the IP address that the 
+   Kitematic instance is proxying the CDAP instance. The easiest way to do that is click
+   on the *Settings* tab, and then the *Ports* tab:
+ 
+     .. image:: ../../_images/kitematic/kitematic-5-links.png
+        :width: 8in
+        :align: center
+
+
+#. This shows that the CDAP instance is listening on Docker port ``9999``, and the Kitematic 
+   instance is listening on ``192.168.99.1100:32769``. The text in blue is a link;
+   clicking it will open it in your system web browser and connect to the CDAP UI:
```

Comment:
Can say "This shows that the CDAP instance is listening on the internal port `9999` within the Docker instance, and exposes that port on the external IP address and port `192.168.99.1100:32769`.